sdk/communication/communication-chat/review/communication-chat.api.md

```diff
@@ -76,50 +75,69 @@ export interface ChatClientOptions extends PipelineOptions {
 }
 
 // @public
-export interface ChatMessage extends Omit<RestChatMessage, "senderId" | "content"> {
+export interface ChatMessage {
     content?: ChatMessageContent;
-    sender?: CommunicationUserIdentifier;
+    createdOn: Date;
+    deletedOn?: Date;
+    editedOn?: Date;
+    id: string;
+    senderCommunicationIdentifier?: CommunicationIdentifierModel;
+    senderDisplayName?: string;
+    sequenceId: string;
+    type: ChatMessageType;
+    version: string;
 }
 
-// @public (undocumented)
-export interface ChatMessageContent extends Omit<RestChatMessageContent, "participants"> {
+// @public
+export interface ChatMessageContent {
+    initiatorCommunicationIdentifier?: CommunicationIdentifierModel;
+    message?: string;
     participants?: ChatParticipant[];
+    topic?: string;
```

Comment:
Isn't topic also a thread property and not for a message? This whole `ChatMessageContent` type seems like it's not a chat message content but models a  chat thread.

Reply:
@juancamilor  @amrelroumy-msft

Reply:
Another system message - this one's for `topicUpdate` type messages

Reply:
I remember our conversations around system messages now. A better way to solve this would be to have seperate types like `ChatMessageContent` and `SystemChatMessageContent` and the SDK will return `ChatMessageContent | SystemChatMessageContent` where both are applicable.

We can make it a discriminated union even and add a `kind` property.

Reply:
@DominikMe are you suggesting this just for JS ? We reviewed this already, hard to keep adding changes without pushing GA now. Also it will be very different from the other sdks. I don't think we should do this now.